MetaMask/metamask-mobile
# test: initial ramps E2E test to build quote

## **Description**

This is the initial foundation for Ramps Mobile E2E tests. This adds the first basic user flow, which is to login, go through onboarding, select region and payment method, and verify the Build Quote view appears. 

## **Related issues**

https://consensyssoftware.atlassian.net/browse/RAMPS-1868
https://consensyssoftware.atlassian.net/browse/RAMPS-1870

## **Manual testing steps**
```
 PASS  e2e/specs/ramps/onramp.spec.js (43.097 s)
  SmokeRamps Buy Crypto
    ✓ should select Region and Payment Method to see the Build Quote screen (15625 ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        43.154 s
```


## **Screenshots/Recordings**

https://github.com/user-attachments/assets/30eed114-5f32-4989-9b07-65ec818c0381


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
